### ISSUE TITLE
Crash when Project's path includes Hebrew characters

### ISSUE THREAD

**Issue by aizenimr** [olivierkes/manuskript#312]
This problem originated during discussion on Issue #279 .
The project kept crashing under Windows platform but not under Linux, and I mistakenly thought that it is somehow connected to Spell checker installation issue.
However, the problem stopped when I moved the project to a folder where its path had no Hebrew characters at all, and also renamed the project's library and main filename to English title. After that the problem was solved - it stopped crashing under Windows.
This problem does not happen when using Linux.

**Comment by gedakc** [comment]
On Windows can you try manually creating the folder with the Hebrew characters using either the Windows file explorer or the command prompt?

If this fails to work properly then that would suggest that the issue is specific to Windows, and not just Manuskript.

**Comment by aizenimr** [comment]
Yes, I can create the folder using the explorer with Hebrew characters (or a mix between Hebrew and English characters).
Furthermore, it seems that the problem happens when the folder has Hebrew characters, but the MSK file itself can be with Hebrew characters without any problem.

**Comment by aizenimr** [comment]
Just to clarify, I don't think it is much of a big deal. Probably some warning in the wiki or proper error message for Windows user should suffice until this issue is resolved.

**Comment by gedakc** [comment]
How long is the longest path when loading the manuskript project?

To capture this information you can open a command prompt, change to the directory containing manuskript.exe, and then run `manuskript.exe` and load the project which causes the crash.  Then copy and paste the output into this issue for us to see.

The reason I ask is that you might have encountered a [maximum path limitation of 260 characters in Windows](https://msdn.microsoft.com/en-us/library/windows/desktop/aa365247(v=vs.85).aspx#maxpath).

**Comment by gedakc** [comment]
This issue might be related or a duplicate of issue #313.
